### ISSUE TITLE
2.7.0 issue: path name is id instead of the file path

### ISSUE THREAD

**Issue by stevez** [iFaxity/vite-plugin-istanbul#34]
plugin version : 2.7.0
vite version: 2.9.1

1. first console get the warning message:
[WebServer] [plugin:vite:istanbul] context method getFileName() is not supported in serve mode. This plugin is likely not vite-compatible.
2. when you open the generated json file, it looks like:
```
 "1649799163381.js": {
        "path": "1649799163381.js",
        "statementMap": {
            "0": {
                "start": {
                    "line": 1,
                    "column": 19
                },
                "end": {
                    "line": 1,
                    "column": 82
                }
            },
            "1": {
                "start": {
                    "line": 7,
                    "column": 0
                },
                "end": {
                    "line": 17,
                    "column": 43
                }
            }
        },
        "fnMap": {},
        "branchMap": {},
        "s": {
            "0": 1,
            "1": 1
        },
        "f": {},
        "b": {},
        "inputSourceMap": {
            "version": 3,
            "mappings": ";AAAA;AACA;AACA;AACA;;AAEAA,SAASC,OACP,wBAAC,MAAM,YAAP;AAAA,YACE,wBAAC,KAAD;AAAA;AAAA;AAAA;AAAA;AADF;AAAA;AAAA;AAAA;AAAA,UAGAC,SAASC,eAAe,MAAxB,CAJF",
            "names": [
                "ReactDOM",
                "render",
                "document",
                "getElementById"
            ],
            "sources": [
                "/Users/s5474013/github/playwright-test-coverage/src/index.tsx"
            ],
            "file": "/Users/s5474013/github/playwright-test-coverage/src/index.tsx"
        },
        "_coverageSchema": "1a1c01bbd47fc00a2c39e90264f33305004495a9",
        "hash": "6165b6380d2c81cf1ff3baff8f189b45479f31c9"
    }
```
3. please run the code in this repo for verification: 
https://github.com/stevez/playwright-test-coverage/tree/integrate-vite-plugin-istanbul
branch: integrate-vite-plugin-istanbul
the json files are in the .nyc_output

4. the issue might related to this line:
https://github.com/iFaxity/vite-plugin-istanbul/blob/main/src/index.ts#L126
It might not be compatible with vite 2.9.1

**Comment by iFaxity** [comment]
Hi, this was problem was introduced by #27 and a fix is now available as version 2.7.1

**Comment by stevez** [comment]
yes it works
